provisioning/stream/kubernetes/templates/cloud/azure/service-http-source.yaml

```diff
@@ -0,0 +1,19 @@
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: stream-http-source
+  namespace: $NAMESPACE
+  labels:
+    app: stream-http-source
+  annotations:
+    service.beta.kubernetes.io/azure-load-balancer-internal: "true"
```

Comment:
Azure cloud has no `ingress` defined here, there is some other method, somewhere else, when it is set.

The `app` label must match, ... see [SLACK](https://keboolaglobal.slack.com/archives/C055WKPHYSD/p1717077126190119?thread_ts=1717076471.004419&cid=C055WKPHYSD)

So, can we keep `app=stream-api`, `app=stream-http-source` (for consistency with our other definitions). 
Or should we modify it to `app=stream`, `app=stream-in`?

Reply:
Thanks!